smb_slam/launch/localization_open3dslam.launch

```diff
@@ -2,14 +2,14 @@
 <launch>
 
   <!-- Arguments -->
+  <!-- Make sure to Set the correct absolute Map path in $(find smb_slam)/config/open3d_slam/param_rs16_localization.lua -->
+  <!-- Without setting the map path, it will throw an error-->
   <arg name="cloud_topic"           default="/rslidar_points"/>
+  <arg name="parameter_filename"    default="param_rs16_localization.lua"/>
   <arg name="parameter_folder_path" default="$(find smb_slam)/config/open3d_slam/"/>
-  <arg name="parameter_filename"    default="params_robosense_rs16.yaml"/>
-  <arg name="read_from_rosbag"      default="false"/>
-  <arg name="rosbag_filepath"       default=""/> # Can be used to process rosbag sequentially, usually much faster without missing messages
   <arg name="launch_rviz"           default="false"/>
   <arg name="launch_sensors"        default="false"/>
-  <arg name="use_sim_time"          default="false"/>
```

Comment:
one could potentially run the localization in sim. Don't we need this argument then?

Reply:
The sim time argument is only used if the rosbag file is run through the launch file. 
For the localization mode, we don't provide an option to run the rosbag file from the same launch file (Since this will be tested live). Alternatively one can still run the rosbag file for localization from a separate terminal so I don't think this is necessary.